<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1304</width>
    <height>699</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>JNK OpenFOAM IDE</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>settings/Icon.png</normaloff>settings/Icon.png</iconset>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="pth">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>0</y>
      <width>621</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Open controlDict file</string>
    </property>
   </widget>
   <widget class="QTreeView" name="treeView">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>401</width>
      <height>571</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="rld">
    <property name="geometry">
     <rect>
      <x>1060</x>
      <y>610</y>
      <width>89</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>Reload</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="tln">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>610</y>
      <width>781</width>
      <height>25</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="tlbl">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>610</y>
      <width>71</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Terminal</string>
    </property>
   </widget>
   <widget class="QPushButton" name="tRn">
    <property name="geometry">
     <rect>
      <x>890</x>
      <y>610</y>
      <width>89</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>Run</string>
    </property>
   </widget>
   <widget class="QPushButton" name="svBtn">
    <property name="geometry">
     <rect>
      <x>1160</x>
      <y>610</y>
      <width>89</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>Save</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1304</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionOpen"/>
    <addaction name="actionNew_File"/>
    <addaction name="actionSave"/>
    <addaction name="dlt"/>
    <addaction name="actionExit"/>
    <addaction name="intd"/>
   </widget>
   <widget class="QMenu" name="menuRun">
    <property name="title">
     <string>Run</string>
    </property>
    <addaction name="actionreactingFoam"/>
    <addaction name="rPara"/>
    <addaction name="pv"/>
    <addaction name="separator"/>
    <addaction name="actionBlock_Mesh"/>
    <addaction name="chMesh"/>
    <addaction name="actionDecompose"/>
    <addaction name="actionReconstruct"/>
    <addaction name="separator"/>
    <addaction name="actionSettings"/>
    <addaction name="separator"/>
    <addaction name="cPar"/>
    <addaction name="cAl"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuRun"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionOpen">
   <property name="checkable">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Open</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+O</string>
   </property>
   <property name="menuRole">
    <enum>QAction::ApplicationSpecificRole</enum>
   </property>
  </action>
  <action name="actionExit">
   <property name="text">
    <string>Exit</string>
   </property>
  </action>
  <action name="actionreactingFoam">
   <property name="text">
    <string>Solve</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+R</string>
   </property>
  </action>
  <action name="rPara">
   <property name="text">
    <string>Solve Parallel</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Shift+R</string>
   </property>
  </action>
  <action name="actionBlock_Mesh">
   <property name="text">
    <string>Block Mesh</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+B</string>
   </property>
  </action>
  <action name="actionDecompose">
   <property name="text">
    <string>Decompose</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+D</string>
   </property>
  </action>
  <action name="actionReconstruct">
   <property name="text">
    <string>Reconstruct</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+P</string>
   </property>
  </action>
  <action name="actionSettings">
   <property name="text">
    <string>Configurations</string>
   </property>
  </action>
  <action name="actionNew_File">
   <property name="text">
    <string>New File</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+N</string>
   </property>
  </action>
  <action name="dlt">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Delete Selected File</string>
   </property>
   <property name="shortcut">
    <string>Del</string>
   </property>
  </action>
  <action name="cPar">
   <property name="text">
    <string>Clean Processor Dir's</string>
   </property>
  </action>
  <action name="cAl">
   <property name="text">
    <string>Clean All</string>
   </property>
  </action>
  <action name="pv">
   <property name="text">
    <string>Para View</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+W</string>
   </property>
  </action>
  <action name="rfs">
   <property name="text">
    <string>Refresh</string>
   </property>
   <property name="shortcut">
    <string>F5</string>
   </property>
  </action>
  <action name="actionSave">
   <property name="text">
    <string>Save Current File</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+S</string>
   </property>
  </action>
  <action name="chMesh">
   <property name="text">
    <string>Check Mesh</string>
   </property>
  </action>
  <action name="intd">
   <property name="text">
    <string>Auto Indent</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+I</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
